tests/performance-tests/Dockerfile-node-chromium

```diff
@@ -0,0 +1,7 @@
+FROM node:18
```

Comment:
```suggestion
FROM node:16
```

Reply:
Sure thing!  I guess this aligns with other Node versions we use elsewhere?

Reply:
Yeah, that's right. Also since Node 18 is a bit of a high version to be on at the moment (the majority of things are a bit early to be adopted).